server/index.js

```diff
@@ -170,6 +186,31 @@ module.exports = config => {
         .listen(port, () => {
           logger.info('bpanel app running on port', port);
         });
+
+      if (process.env.BPANEL_SERVE_HTTPS) {
```

Comment:
I think we might be able to use the new bcfg setup to make this more flexible. We could grab `serve-https` from the configs which could be passed by env or conf file.